src/Microsoft.Health.Fhir.Api/Features/Binders/PartialDateTimeBinder.cs

```diff
@@ -27,8 +28,9 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
 
                     bindingContext.Result = ModelBindingResult.Success(result);
                 }
-                catch
+                catch (FormatException ex)
```

Comment:
Should we still have a universal exception catch just in case, or let it bubble up?

Reply:
I'm not big fan of blank policies. They tend to hide mistakes from you.
Right now only exception we intent to throw during parse is FormatException, and if something would go wrong and I put catch for Exception, we will never see it in logs, therefore we would be blind to something going on.

As an option I can probably plumb logger here and log critical exception, but I'm too lazy for that.